spec/game_looper_spec.rb

```diff
@@ -21,71 +21,128 @@ def initialize(marker)
 end
 
 class TestBoard
-  def mark_space(_message, _error_message); end
-end
+  attr_reader :values
+  attr_accessor :move_history
 
-class TestDisplay end
+  def initialize(values: nil)
+    @values = values
+    @move_history = []
+  end
 
-class TestPrompt
-  def call(_message, _error_message); end
+  def mark_space(token, space)
+    move_history << "Board with #{token} in space #{space}"
+  end
 end
 
 class TestOutcomeCheckerForGameLooper
-  def initialize(board:)
+  attr_accessor :board, :win_values, :draw_values
+
+  def initialize(board:, win_values:, draw_values:)
```

Comment:
**suggestion (non-blocker)**: Passing in `win_values` and `draw_values` to control the way the test outcome checker behaves could be finicky and brittle.

Here are a couple of alternatives that might help.

1. Define multiple test doubles for an abstraction. It is ok to have a `AlwaysResultsInDraw` and `WillWin` test doubles that you can use as you see fit. In other words, you don't need to have a single `TestOutcomeChecker` that can work for every test in the spec. You can have as many doubles as you need.
2. Is there a way to simplify the outcome checker? There are at least three states of the game that come to mind. 1.) In Progress 2.) The game is over and it is a draw, 3.) The game is over and there is a winner. Only two of these states are explicit (`win?`, `draw?`). A complicated test double could give us feedback on our design.